addons/website/static/src/scss/configurator.scss

```diff
@@ -84,12 +84,40 @@
                 min-width: 13ch;
             }
 
-            .industry_selection.o_step_todo {
+            .o_configurator_purpose_dd.o_step_todo {
+                min-width: 12ch;
+            }
+
+            .o_configurator_industry_wrapper, .o_configurator_industry_wrapper input {
+                line-height: 1.5;
+                cursor: pointer;
+            }
+
+            .o_configurator_industry_wrapper {
+                position: relative;
+                display: inline-block;
                 min-width: 17ch;
             }
 
-            .o_configurator_purpose_dd.o_step_todo {
-                min-width: 12ch;
+            .o_configurator_industry_wrapper::after {
+                content: attr(data-value) '|';
+            }
+
+            .o_configurator_industry_wrapper input {
+                position: absolute;
+                width: 100%;
+                height: $line-height-base * 1em;
+                padding: 0;
+                border: none;
+                border-bottom: 3px solid theme-color(primary);
+                color: theme-color(primary);
+                font-weight: 500;
+                font-style: italic;
```

Comment:
Should also be applied on the text behind the input otherwise the input is not exactly resized to the size of the text.